clients/geth/specular/Dockerfile

```diff
@@ -1,11 +1,18 @@
 # Compile contracts for binding generation
 FROM node:16-alpine3.17 as contracts-builder
 
-RUN apk add --no-cache make git
+RUN apk add --no-cache make git python3 g++
+RUN npm install -g pnpm
```

Comment:
use `corepack enable` to install `pnpm`; merge two`RUN` command to avoid unnecessary layers

Reply:
nvm, see #120

Reply:
Updated